SocialCareCaseViewerApi.Tests/V1/UseCase/MashReferral/UpdateMashReferralTests.cs

```diff
@@ -62,6 +62,28 @@ public void UpdatingMashReferralThrowsWorkerNotFoundExceptionWhenGetWorkerByWork
                 .WithMessage($"Worker with email \"{request.WorkerEmail}\" not found");
         }
 
+        [Test]
+        public void UpdatingMashReferralUnAssignOnSuccessReturnsMashReferralResponse()
+        {
+            var worker = TestHelpers.CreateWorker();
```

Comment:
I might have missed it but it doesn't look like we use these variable elsewhere yet. Do we need it?

Reply:
Nope we don't